scss/sections/_hero.scss

```diff
@@ -0,0 +1,220 @@
+#hero {
+  display: flex;
+  max-height: 1080px;
+  width: 90%;
+  margin: 0 0 0 10%;
+  flex-direction: row;
+  justify-content: space-between;
+
+  .hero_text {
```

Comment:
eeeeee title is more fit here.
text is to more generic meaning

Reply:
can i replace "text" with "title"?